docs/oadp_monitoring.md

```diff
@@ -0,0 +1,241 @@
+
+
+## OADP Monitoring Setup
+
+Monitoring [metrics](#metrics) requires enabling monitoring for the user-defined projects and creating ServiceMonitor resource to scrape those metrics from the already enabled OADP service endpoint that lives in the `openshift-adp` namespace.
+
+> **Note:** The configuration files that allows to enable User Workload Monitoring can be found in the [docs/examples/manifests/user_monitoring/](./examples/manifests/user_monitoring/) folder of this repository, however they will override any previous configuration that could have been on the cluster, so use them with caution. They are named with the number prefix, which represents the order in which they should be applied.
+
+### Prerequisites
+* OADP operator, a credentials secret, and a DataProtectionApplication (DPA) CR are all created. Follow [these steps](/docs/install_olm.md) for installation instructions.
+* shell with the `oc` CLI command available and admin level access to the OpenShift cluster.
+
+
+### Enable and Configure User Workload Monitoring
+
+This paragraph will provide a short set of instructions how to enable user workload monitoring for an OADP project in the cluster. For comprehensive set of configuration options refer to the [enabling monitoring for user-defined projects](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/enabling-monitoring-for-user-defined-projects#doc-wrapper) documentation.
+
+
+1. Edit the `cluster-monitoring-config` ConfigMap object in the `openshift-monitoring` namespace and add or enable the `enableUserWorkload` option under `data/config.yaml`. 
+
+    ```shell
+    $ oc edit configmap cluster-monitoring-config -n openshift-monitoring
+    ```
+
+
+    ```yaml
+    apiVersion: v1
+    data:
+      config.yaml: |
+        enableUserWorkload: true  # Add this option or set to true
+    kind: ConfigMap
+    metadata:
+    # [...]
+    ```
+
+3. After a short period of time verify the User Workload Monitoring Setup by checking if the following components are up and running in the `openshift-user-workload-monitoring` namespace
+
+    ```shell
+    $ oc get pods -n openshift-user-workload-monitoring
+    NAME                                   READY   STATUS    RESTARTS   AGE
+    prometheus-operator-6844b4b99c-b57j9   2/2     Running   0          43s
+    prometheus-user-workload-0             5/5     Running   0          32s
+    prometheus-user-workload-1             5/5     Running   0          32s
+    thanos-ruler-user-workload-0           3/3     Running   0          32s
+    thanos-ruler-user-workload-1           3/3     Running   0          32s
+    ```
+
+4. Please verify the existence of the `user-workload-monitoring-config` ConfigMap in the `openshift-user-workload-monitoring`, and if it is found, skip the following two steps (5th and 6th) accordingly
```

Comment:
NIT: `4th and 5th` instead of `5th and 6th`

Reply:
Fixed.